api/overview/azure/edgeorder.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Azure Edge Hardware Center libraries for .NET
+description: Reference for Azure Edge Hardware Center libraries for .NET
+ms.date: 02/09/2021
+ms.topic: reference
+ms.service: edgeorder
+---
+# Azure Edge Hardware Center APIs for .NET
+
+## Overview
+
+[Azure Edge Hardware Center](https://docs.microsoft.com/en-us/azure/azure-edge-hardware-center/) is an Azure service that lets you order a variety of first party Azure hardware and discover third party hardware offered by our partners. Edge Hardware Center lets you see and track all your order related information at one place.
+
+## Management APIs
+
+Create and manage Azure Edge Hardware Center orders.
+
+Install the [NuGet package](https://www.nuget.org/packages/Azure.ResourceManager.EdgeOrder) directly from the Visual Studio [Package Manager console](https://docs.microsoft.com/nuget/tools/package-manager-console) or with the [.NET Core CLI](https://docs.microsoft.com/dotnet/core/tools/dotnet-add-package).
+
+#### Visual Studio Package Manager
+
+```powershell
+Install-Package Azure.ResourceManager.EdgeOrder -Version 1.0.0-beta.1
+```
+
+#### .NET Core CLI
+
+```dotnetcli
+dotnet add package Azure.ResourceManager.EdgeOrder --version 1.0.0-beta.1
```

Comment:
```suggestion
dotnet add package Azure.ResourceManager.EdgeOrder --prerelease
```

This form may be helpful, as it will snap to the latest beta and not require doc updates to bump the version.